TWLight/resources/models.py

```diff
@@ -206,6 +206,10 @@ class Meta:
         "description such as collections, instructions, notes, special "
         "requirements, alternate access options, unique features, citations notes."))
         
+    additional_resources = models.TextField('video tutorials', blank=True,
```

Comment:
The naming here bothers me.  We're calling one thing a resource, a tutorial, and a url in various places. Pick one and stick with it. If this is only ever going to be for tutorial video urls, then call the field tutorial_video_urls or something use the built in url validator https://docs.djangoproject.com/en/1.11/ref/validators/#urlvalidator

If you're thinking about being flexible for including resources other than video links, then break this out into its own model, and give it separate fields for url, resource type, and purpose, eg [url, video, tutorial], [url, text, faq], etc.

Reply:
Regarding the URL validator - the idea was that this field could accept a comma separated list of videos. Would the validator work in this case?

Reply:
oh, yeah, that's icky. You'd then be splitting that string down the road, right? That's basically disregarding Django as an ORM.  If you want multiple things, you should have multiple objects. This should be its own model.

Reply:
That would indeed make more sense.